inquire/src/ui/render_config.rs

```diff
@@ -497,6 +501,7 @@ pub mod calendar {
                 today_date: StyleSheet::empty().with_fg(Color::LightGreen),
                 different_month_date: StyleSheet::empty().with_fg(Color::DarkGrey),
                 unavailable_date: StyleSheet::empty().with_fg(Color::DarkGrey),
+                weekend: StyleSheet::empty().with_fg(Color::DarkRed),
```

Comment:
```suggestion
                weekend: StyleSheet::empty().with_fg(Color::DarkGrey),
```

<img width="189" alt="image" src="https://github.com/mikaelmello/inquire/assets/17488630/652115a1-cdd6-4523-9160-38c30946910a">
<img width="178" alt="image" src="https://github.com/mikaelmello/inquire/assets/17488630/67e752a3-71ab-44d8-9ca2-2c89b1759d1f">

It's not that I necessarily defend grey as being the ultimate UX choice, but I think the red implies some information in a way, as if it's an unavailable date.